app/services/catalog_remediator.rb

```diff
@@ -11,6 +13,8 @@ def initialize(druid:, version:)
     @version = version
   end
 
+  # prunes ZipParts and ZippedMoabVersion from database for any ZippedMoabVersions that have replication errors for
+  #   the druid and version
```

Comment:
Can you say more about having similar but slightly different documentation on lines 5-6 and 16-17 (and also line 3)? I might be biased in favor of a single comment at the class-level since this service object only does one thing.

Reply:
happy to have it on the instance method only;  not sure the class will continue to do only one thing.

Reply:
now on instance method only.  thanks for suggestion.